source/features/useful-not-found-page.tsx

```diff
@@ -36,54 +38,59 @@ function parseCurrentURL(): string[] {
 	return parts;
 }
 
-// If the resource was deleted, link to the commit history
-async function addCommitHistoryLink(bar: Element): Promise<void> {
-	const parts = parseCurrentURL();
-	parts[2] = 'commits';
-	const url = '/' + parts.join('/');
-	if (await is404(location.origin + url)) {
+async function getLatestChangeToFile(): Promise<Record<string, any> | void> {
+	const url = new GitHubURL(location.href);
+	const {branch, filePath} = url;
```

Comment:
I think it's not used anywhere else
```suggestion
	const {branch, filePath} = new GitHubURL(location.href);
```